mesheryctl/internal/cli/root/system/channel.go

```diff
@@ -189,9 +189,16 @@ var switchCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		err = restartCmd.RunE(cmd, args)
-		if err != nil {
-			return err
+		//Refresh meshery to reflect channel update
+
+		//stop meshery
+		if err := stop(); err != nil {
+			return errors.Wrap(err, utils.SystemError("Failed to stop Meshery"))
+		}
+
+		// Start meshery
+		if err := start(); err != nil {
+			return errors.Wrap(err, utils.SystemError("Failed to start Meshery"))
```

Comment:
Let's change the `restart` behaviour of Kubernetes platform to `stop` and `start` as well similar to the Docker platform. That would be the better change here I guess. More straightforward. A more consistent behaviour. 

We can remove the platform check in `restart` and do the above mentioned step.

Reply:
Issue mentioned [here](https://github.com/meshery/meshery/issues/2956) with `system reset` is only with docker.
After updating reset for k8s will it be a problem a problem with manifests?

Reply:
We do not have to update `reset`. We only have to update the `restart` command. So instead of having to change the call to `restart` to `stop` and `start` like you did here, we can just call `restart`. Does that make sense?